vega_sim/api/trading.py

```diff
@@ -547,7 +547,6 @@ def order_submission(
         type=order_type,
         reduce_only=reduce_only,
         post_only=post_only,
-        iceberg_opts=iceberg_opts,
```

Comment:
Why is this being deleted?

Reply:
We were already setting the opts a few lines below with this didn't seem worth the duplication.
```
    if iceberg_opts is not None:
        command.iceberg_opts.CopyFrom(iceberg_opts)
```